internal/vcs/metadata.go

```diff
@@ -592,6 +598,79 @@ func (f *metadataFetcher) getAtlantisMetadata(path string) (Metadata, error) {
 	return m, nil
 }
 
+// getTFCMetadata returns Metadata built from a TFC request: https://www.terraform.io/cloud-docs/api-docs/run-tasks/run-tasks-integration#request-body
+// We cannot rely on any local git information we run this process in the Infracost Cloud. All OS variables are populated
+// by the Infracost Cloud run task worker, which passes them as environment flags when running the Infracost CLI.
+func (f *metadataFetcher) getTFCMetadata(path string) (Metadata, error) {
+	remote := urlStringToRemote(os.Getenv("TFC_VCS_URL"))
```

Comment:
Since we're creating these env vars why not follow the same format as the existing `INFRACOST_VCS_` ones?
`INFRACOST_VCS_PULL_REQUEST_URL`
`INFRACOST_VCS_REPOSITORY_URL`
...

Reply:
Yeah can do, thought it would be more explicit - but it makes sense to reuse existing ones. Ill fixup now

Reply:
I think there's maybe two other reasons to use `INFRACOST_VCS_` for these:
1. It scopes it to Infracost, so we make sure we don't ever conflict with other vars.
2. If we have generic `INFRACOST_VCS_` env vars we can use them for any deployment system to integrate with Infracost Cloud.
What do you think?

Reply:
resolved in https://github.com/infracost/infracost/pull/1934/commits/0fd966d2375e72c8c97f158857920ca8849df8ad